src/sync-merkleroots.test.ts

```diff
@@ -0,0 +1,315 @@
+import fetchMock from 'jest-fetch-mock';
+import {
+  ErrorStaleLastEvaluatedKey,
+  ErrorSyncMerkleRootsTimeout,
+  ExclusiveStartKeyPage,
+} from '../dist/typescript-npm-package.cjs';
+import { SpvWalletClient } from './client';
+import { MerkleRoot, MerkleRootsRepository } from './types';
+
+const server = 'http://localhost:3003/api/v1';
+
+const client = new SpvWalletClient(server, {
+  xPriv:
+    'xprv9s21ZrQH143K3axKPtYBDKsrAvN3J85z6nZuW5ihYK8JpDWfqHdjswvUnbXzUpMBne1WD6FQmTzymB4Pt3u3UVSauxzq5PswBWr3vYtowmW',
+});
```

Comment:
Consider to create new client instances for every single test separately. To ensure one test doesn't influence to another.